packages/verkle/src/types.ts

```diff
@@ -60,7 +60,7 @@ export interface Point {
 
 export type Proof = Uint8Array[]
 
-export interface VerkleTrieOpts {
+export interface VerkleTreeOpts {
```

Comment:
I have no strong opinion about this, but just noting: our MPT is a merkle Trie, so the Trie would be more in-line with that. But I don't mind changing it to a Tree.

Reply:
Actually the technical term is `Tree` (Merkle Tree), so it would actually make sense to rename this to a `Tree` :smile:

Reply:
Yes we discussed that a bit yesterday, and decided to go with Tree. The "trie" terminology (even though technically correct for data structures where to retrieve a value for a given key, you follow the "path" of that key from node to node) tends to confuse people anyway